frontend/src/components/ant-design/Tag/Tag.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {
+  CheckCircleOutlined,
+  ClockCircleOutlined,
+  CloseCircleOutlined,
+  ExclamationCircleOutlined,
+  MinusCircleOutlined,
+  SyncOutlined,
+} from '@ant-design/icons/lib';
+import { Box } from '@components/atoms/Box';
+import { Tag } from './Tag';
+
+export default {
+  component: Tag,
+  title: 'Tag',
+  excludeStories: /.*Data$/,
+};
+
+const log: () => void = () => {
+  alert('Callback executed when tag is closed');
```

Comment:
Use [Storybook Actions Addon](https://github.com/storybookjs/storybook/tree/master/addons/actions) for event handlers testing